cmd/helm/create.go

```diff
@@ -75,6 +75,9 @@ func newCreateCmd(out io.Writer) *cobra.Command {
 			if len(args) == 0 {
 				return errors.New("the name of the new chart is required")
 			}
+			if len(args) > 1 {
+				return errors.New("create one chart at a time")
```

Comment:
Can you perhaps write up a more meaningful error message? This doesn't really explain to the user what the issue was.

Reply:
sure. Sorry for inconvenience. Actually, I was trying to come up with some short message 😛